modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -1697,6 +1699,117 @@ test.describe('Manage object entries through View Object Entries', () => {
 		});
 	});
 
+	test(
+		'error message is displayed when trying to view a deleted object entry with a user with view-only permissions',
+		{tag: ['@LPD-61276']},
+		async ({apiHelpers, page, viewObjectEntriesPage}) => {
+			let entryUrl: string;
+			let user;
+
+			await test.step('Create object entry and get its URL', async () => {
```

Comment:
This is a setup step. I tend to leave setup steps out of `test.step` calls, or inside a "Setup" step. The step you describe here starts in `await viewObjectEntriesPage.goto(customObject.className)`.